src/js/pensions/config/form.js

```diff
@@ -774,22 +798,169 @@ const formConfig = {
               'view:hasDependents': {
                 type: 'boolean'
               },
+              // merge with definition, provide minItems: 1, and items.required: ['relationship', 'fullName']
               dependents: {
                 type: 'array',
                 minItems: 1,
                 items: {
                   type: 'object',
-                  required: ['relationship', 'fullName'],
+                  required: ['dependentRelationship', 'fullName'],
                   properties: {
-                    relationship: dependents.items.properties.relationship,
+                    dependentRelationship: dependents.items.properties.dependentRelationship,
                     fullName: dependents.items.properties.fullName,
-                    childDateOfBirth: dependents.items.properties.childDateOfBirth
+                    childDateOfBirth: dependents.items.properties.childDateOfBirth,
+                  }
+                }
+              }
+            }
+          }
+        },
+        childrenInformation: {
+          path: 'household/dependents/children/information/:index',
+          title: item => `${item.fullName.first} ${item.fullName.last} information`,
+          depends: hasChildren,
```

Comment:
I think you can leave this off. We shouldn't generate any of these pages if dependents array combined with the item filter results in an empty array.